GoogleMobileAds-HyBid-Adapters.podspec

```diff
@@ -0,0 +1,49 @@
+Pod::Spec.new do |spec|
+
+  spec.name         = "GoogleMobileAds-HyBid-Adapters"
+  spec.version      = "0.1.0"
+  spec.summary      = "Test iOS adapter for Google Mobile Ads"
+  spec.description = <<-DESC
+                     Supported ad formats:
+                     GAM: Banner, MRect, Leaderboard, Interstitial
```

Comment:
I'd add rewarded and native here since the Admob adapters work as well for DFP mediation : )

Reply:
ah really @eros902002 ? cool 👍 so if that's the case then we don't need that differentation between GAM and GAD since both has the same formats?

Reply:
Yes exactly. We can differentiate between header bidding and mediation but for the latter one it's the case that it works the same on both Admob and DFP

Reply:
cool 👍 i've unified the description @eros902002